src/rc/checkpath.c

```diff
@@ -69,6 +70,37 @@ const char * const longopts_help[] = {
 };
 const char *usagestring = NULL;
 
+/*
+ * Work around limitation in Linux when calling fchmod() on a file
```

Comment:
can you add a bit more detail (and perhaps include a link to man7.org) here ?  i know what limitation you're referring to, but for people who aren't into these low level esoteric details, this comment is just mysterious.